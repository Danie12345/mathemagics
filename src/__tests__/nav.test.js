import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import { BrowserRouter } from 'react-router-dom';

const Nav = require('../components/nav/Nav').default;

test('loads the nav with everything', () => {
  const components = <BrowserRouter><Nav /></BrowserRouter>;
  const tree = renderer.create(components).toJSON();
  expect(tree).toMatchSnapshot();
});
