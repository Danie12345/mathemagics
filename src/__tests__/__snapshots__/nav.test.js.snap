// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loads the container with everything 1`] = `
<nav
  className="navBar"
>
  <h1>
    Mathemagics
  </h1>
  <ul>
    <li>
      <a
        className=""
        href="/home"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        className=""
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        className=""
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;

exports[`loads the nav with everything 1`] = `
<nav
  className="navBar"
>
  <h1>
    Mathemagics
  </h1>
  <ul>
    <li>
      <a
        className=""
        href="/home"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        className=""
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        className=""
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;

exports[`that home works 1`] = `
<nav
  className="navBar"
>
  <h1>
    Mathemagics
  </h1>
  <ul>
    <li>
      <a
        className=""
        href="/home"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        className=""
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        className=""
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
